Assets/Scripts/PlayerAvatar.cs

```diff
@@ -8,6 +8,7 @@ public class PlayerAvatar : MonoBehaviour {
     private BoxCollider2D boxCollider;
     private bool grounded;
     private int hitpoints;
+    private bool isDead;

```

Comment:
Is this just a proxy for hitpoints == 0?

Reply:
replaced with a function https://github.com/CleverGameDev/Cleverfall/pull/28/commits/ab3ee369169955d34296ebf4c3e18fd0b3449bd1